DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -149,7 +149,8 @@
    <version ClassVersion="10" checksum="2692173055"/>
   </class>
   <class name="std::vector<pat::tau::TauCaloSpecific>" />
-  <class name="pat::tau::TauPFEssential" ClassVersion="11">
+  <class name="pat::tau::TauPFEssential" ClassVersion="12">
+    <version ClassVersion="12" checksum="1052833547" />
```

Comment:
changes in this file should match changes in 81X in full (to include classVersion=13)